### PR TITLE
fix(#2056): date picker misc docs updates

### PR DESCRIPTION
**Pull request by syedszeeshan**
This PR depends on https://github.com/GovAlta/ui-components/pull/2093

PR covers
-date picker reset example
-relative prop added to the properties accordion

### CODE REVIEW

**Comment by lizhuomeng71** [comment]
this issue is verified on react and angular

File: src/routes/components/DatePicker.tsx [hunk 1]
```diff
@@ -175,6 +205,106 @@ export default function DatePickerPage() {
...
+                      </>
+                    );
+                  }
+              `}
+              />
+
+              <CodeSnippet
+                lang="typescript"
+                tags="angular"
+                allowCopy={true}
```
[the remark this answers is not in the document]
Comment (syedszeeshan) [reply src/routes/components/DatePicker.tsx:262]: Thanks, both Reactive and Event based examples have been added.

File: src/routes/components/DatePicker.tsx [hunk 2]
```diff
@@ -175,6 +205,144 @@ export default function DatePickerPage() {
...
+              <CodeSnippet
+                lang="html"
+                tags="angular"
+                allowCopy={true}
+                code={`
+                  <goa-form-item label="Item">
+                    <goa-date-picker (_change)="onChange($event)" name="item" [value]="item"></goa-date-picker>
+                  </goa-form-item>
+
+                  <br/>
```
[the remark this answers is not in the document]
Comment (syedszeeshan) [reply src/routes/components/DatePicker.tsx]: Removed <br> from the actual code as well as the example code.... and replaced with goa-spacer to keep it simple. If we aboslutely need goa-button-group please let me know.

[the remark this answers is not in the document]
Comment (syedszeeshan) [reply src/routes/components/DatePicker.tsx]: Added `<goa-button-group>` now, Thanks.

**Comment by lizhuomeng71** [comment]
For the Angular Example, Event Base Code Snip, there is few duplicate line added

**Comment by syedszeeshan** [comment]
> For the Angular Example, Event Base Code Snip, there is few duplicate line added
> 
> image



> For the Angular Example, Event Base Code Snip, there is few duplicate line added
> 
> image

Good catch, duplicate lines are removed now.

**Pull request merged**